app-misc/openrgb-plugin-effects/openrgb-plugin-effects-0.9_p20250307.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic qmake-utils
+
+DESCRIPTION="Plugin for OpenRGB with various Effects that can be synced across devices"
+HOMEPAGE="https://gitlab.com/OpenRGBDevelopers/OpenRGBEffectsPlugin"
+
+MY_COMMIT="9234e358b20ad83d3249ad13a8765fc192a965f5"
+MY_NOISE_COMMIT="97e62c5b5e26c8edabdc29a6b0a277192be3746c"
+MY_QCODEEDITOR_COMMIT="a9aab24c7970a38d14bc79939306d9d3ba78cf61"
+SRC_URI="
+	https://gitlab.com/OpenRGBDevelopers/OpenRGBEffectsPlugin/-/archive/${MY_COMMIT}/OpenRGBEffectsPlugin-${MY_COMMIT}.tar.bz2 -> ${P}.tar.bz2
+	https://github.com/SRombauts/SimplexNoise/archive/${MY_NOISE_COMMIT}.tar.gz -> SimplexNoise-2019-12-03.tar.gz
+	https://github.com/justxi/QCodeEditor/archive/${MY_QCODEEDITOR_COMMIT}.tar.gz -> QCodeEditor-2021-08-17.tar.gz
+"
+S="${WORKDIR}/OpenRGBEffectsPlugin-${MY_COMMIT}"
+
+LICENSE="GPL-2 MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=app-misc/openrgb-0.9_p20250524:=
+	dev-qt/qtbase:6[gui,widgets,-gles2-only]
+	media-libs/openal
+"
+DEPEND="
+	${RDEPEND}
+	dev-cpp/nlohmann_json
+"
+
+PATCHES=(
+	"${FILESDIR}/${P}-dep.patch"
+	"${FILESDIR}/${P}-qt6.patch"
```

Comment:
this patch is adding `core5compat` and `opengl` to required Qt6 modules but the ebuild is reflecting neither of it